src/Clinic.API/Controllers/OfficesController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AutoMapper;
+using Clinic.API.ViewModels.Office;
+using Clinic.BLL.Interfaces;
+using Clinic.BLL.Models;
+using Microsoft.AspNetCore.Mvc;
+using static Clinic.DAL.Entities.EnumsEntity;
+
+namespace Clinic.API.Controllers
+{
+    [Route("api/offices")]
```

Comment:
[Route("api/[controller]")]?

Reply:
Here i just don't like upper case like api/Offices. Should i change it to standart [Route("api/[controller]")]?